docs/EN_US/ECLStandardLibraryReference/SLR-Mods/DfuPlusExec.xml

```diff
@@ -23,36 +23,39 @@
         <row>
           <entry><emphasis>commandline</emphasis></entry>
 
-          <entry>A null-terminated string containing the DFUPlus.exe command
-          line to execute. The valid arguments are documented in the Client
-          Tools manual, in the section describing the DfuPlus.exe
+          <entry>A null-terminated string containing the DFUPlus command line
+          to execute. The valid arguments are documented in the Client Tools
+          manual, in the section describing the Command Line DFU
           program.</entry>
         </row>
       </tbody>
     </tgroup>
   </informaltable>
 
   <para>The <emphasis role="bold">DfuPlusExec</emphasis> action executes the
-  specified <emphasis>commandline</emphasis> just as the DfuPLus.exe program
-  would do. This simply allows you to have all the functionality of the
-  DfuPLus.exe program available within your ECL code.</para>
+  specified <emphasis>commandline</emphasis> just as the DfuPlus executable
+  program would. This allows you to have all the functionality of DfuPlus
+  available within your ECL code.</para>
+
+  <para>The server= parameter for DfuPlus should almost always be omitted,
```

Comment:
That begs the question when should it be provided?

Better would be:

The server= parameter ... should only be provided if ...